apps/docs/src/app/core/component-docs/table/examples/table-toolbar-example.component.ts

```diff
@@ -54,16 +75,16 @@ export class TableToolbarExampleComponent implements OnInit {
 
     searchInputChanged(event: string): void {
         if (event) {
-            this.displayedRows = this.tableRows.filter(row => {
+            this.displayedRows = this.getSelectedVariantRows().filter(row => {
                 return JSON.stringify(row).toLowerCase().indexOf(event.toLowerCase()) !== -1;
```

Comment:
You are sorting here also based on object properties names, which causes filtering bugs (try to filter by '1')

Reply:
It would be also cool to have such base filtering login inside the table, but thats probably the topic for future discussion